brainframe_qt/ui/main_window/activities/stream_configuration/stream_configuration.py

```diff
@@ -647,7 +647,7 @@ def _display_ip_camera_help(self):
 
     def _handle_send_stream_conf_error(self, exc: BaseException) -> None:
 
-        message_title = self.tr("Error Opening Stream")
+        message_title = self.tr("Error Configuring Stream")
```

Comment:
This changes the translated string, but there are no changes to the translation file

Reply:
Oh I see you have a TODO item on the PR description